From 64c250e60871ff3939c3e4a0101b0a0023cb8e42 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Thu, 28 Jun 2012 16:27:40 +0800
Subject: [PATCH 298/502] ENGR00214150 android ICS run on mx6sl board with 2D
 acceleration enable

android ICS run on mx6sl board with 2D acceleration enable
gpu 2D hwcompser not work without FSL_IMX_DISPLAY define

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 mx5x/libgralloc/Android.mk        | 2 ++
 mx6/libgralloc_wrapper/Android.mk | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/mx5x/libgralloc/Android.mk b/mx5x/libgralloc/Android.mk
index 6a4d489..e6812e7 100755
--- a/mx5x/libgralloc/Android.mk
+++ b/mx5x/libgralloc/Android.mk
@@ -45,6 +45,8 @@ endif
 
 ifeq ($(HAVE_FSL_IMX_IPU),true)
 LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
+else ifeq ($(HAVE_FSL_IMX_GPU),true)
+LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
 endif
 
 LOCAL_MODULE_TAGS := eng
diff --git a/mx6/libgralloc_wrapper/Android.mk b/mx6/libgralloc_wrapper/Android.mk
index 087f497..385d63a 100755
--- a/mx6/libgralloc_wrapper/Android.mk
+++ b/mx6/libgralloc_wrapper/Android.mk
@@ -40,6 +40,8 @@ LOCAL_C_INCLUDES = external/expat/lib
 
 ifeq ($(HAVE_FSL_IMX_IPU),true)
 LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
+else ifeq ($(HAVE_FSL_IMX_GPU),true)
+LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
 endif
 
 LOCAL_MODULE_TAGS := eng
-- 
1.8.0

